src/main/webapp/search-meal-script.js

```diff
@@ -16,6 +16,9 @@ function searchMeal() {
   const queryString = window.location.search;
   const urlParams = new URLSearchParams(queryString);
   const searchLine = urlParams.get("query") ?? "";
+
+  console.log(searchLine);
```

Comment:
If you want to, you can keep this log in production code, IF you add some message to it, such as "Input query: ".
You can also remove this log altogether.